### PR TITLE
[Elements] Update Color structure

### LINKED ISSUE #88: [Elements] Update Color structure
Update colors structure in the `elements` package to combine different shades under a parent key 

- [x] Create a structure like `colors.primary.base` where base is a color string like `#CC10AA` 
- [x] Move existing accents|shades in the parent. e.g. `colors.primaryPlus1` becomes `colors.primary.plus1` 
- [x] Update existing color usages in `tokens`

### PR DESCRIPTION
**Pull request by zarkoboyadjiev-zz**
Closes #88

### CODE REVIEW

File: packages/components/react-web/.storybook/theme.js [hunk 1]
```diff
@@ -4,13 +4,13 @@ import * as elements from '@compassion-gds/elements';
 export default create({
   base: 'light',
 
-  colorPrimary: elements.color.primary,
+  colorPrimary: elements.color.primary.base,
```
Comment (kidroca) [inline packages/components/react-web/.storybook/theme.js]: There should be no `.base` color, we decided to use `.main`

File: packages/components/tokens/components/button.js [hunk 2]
```diff
@@ -7,11 +7,11 @@ module.exports = {
...
     fontSize: { value: `${elements.typography.fontSizes.base}` },
     minHeight: { value: `${elements.size.plus4}` },
-    background: { value: `${elements.color.white}` },
-    color: { value: `${elements.color.primary}` },
+    background: { value: `${elements.color.white.base}` },
+    color: { value: `${elements.color.primary.base}` },
     borderWidth: { value: `1px` },
     borderStyle: { value: `solid` },
-    borderColor: { value: `${elements.color.primary}` },
+    borderColor: { value: `${elements.color.primary.base}` },
```
Comment (kidroca) [inline packages/components/tokens/components/button.js]: There should be no `.base` color, we decided to use `.main`

File: packages/components/tokens/components/checkbox.js [hunk 3]
```diff
@@ -6,31 +6,31 @@ const checkmarkSvg =
...
-      background: { value: `${elements.color.gray}` },
+      borderColor: { value: `${elements.color.gray.base}` },
+      background: { value: `${elements.color.gray.base}` },
       backgroundImage: { value: `url("${checkmarkSvg}")` },
     },
     checked: {
-      borderColor: { value: `${elements.color.primary}` },
-      background: { value: `${elements.color.primary}` },
+      borderColor: { value: `${elements.color.primary.base}` },
+      background: { value: `${elements.color.primary.base}` },
```
Comment (kidroca) [inline packages/components/tokens/components/checkbox.js]: There should be no `.base` color, we decided to use `.main`

File: packages/components/tokens/components/button.js [hunk 4]
```diff
@@ -45,8 +45,8 @@ module.exports = {
   },
 
   primary: {
-    background: { value: `${elements.color.primary}` },
-    color: { value: `${elements.color.white}` },
+    background: { value: `${elements.color.primary.base}` },
+    color: { value: `${elements.color.white.base}` },
```
Comment (kidroca) [inline packages/components/tokens/components/button.js]: There should be no `.base` color, we decided to use `.main`

**Review by kidroca — CHANGES_REQUESTED** [review]
There are a lot of rendering differences compared to before.
#### Before | After
image
image


Some items like Button > Disabled are not even usable:
image

Looking at the code it seems to be because `.base` is used, while the color objects have `.main`

**Comment by kidroca** [comment]
_Clicking "Details" here should navigate you to see the before/after changes_
image

File: packages/components/react-web/.storybook/theme.js (hunk 1 above)
Comment (zarkoboyadjiev-zz) [reply packages/components/react-web/.storybook/theme.js]: Fixed!

File: packages/components/tokens/components/button.js (hunk 2 above)
Comment (zarkoboyadjiev-zz) [reply packages/components/tokens/components/button.js]: Fixed!

File: packages/components/tokens/components/checkbox.js (hunk 3 above)
Comment (zarkoboyadjiev-zz) [reply packages/components/tokens/components/checkbox.js]: Fixed!

File: packages/components/tokens/components/button.js (hunk 4 above)
Comment (zarkoboyadjiev-zz) [reply packages/components/tokens/components/button.js]: Fixed!

**Review by kidroca — CHANGES_REQUESTED** [review]
Logo component colors are not getting picked up correctly: https://deploy-preview-97--competent-einstein-f61f1f.netlify.app/?path=/story/global-design-system-logo--viewport-based-sizing&args=color:brand&globals=measureEnabled:false

**Pull request merged**